sandbox/src/mocks/r4/createServiceRequest/requests/createIncomplete.json

```diff
@@ -0,0 +1,87 @@
+{
+  "resourceType": "ServiceRequest",
+  "subject": {
+    "identifier": {
+      "system": "https://fhir.nhs.uk/Id/nhs-number",
+      "value": "9912003888"
+    }
+  },
+  "category": [
+    {
+      "coding": [
+        {
+          "system": "https://fhir.nhs.uk/CodeSystem/message-category-servicerequest",
+          "code": "referral"
+        }
+      ]
+    }
+  ],
+  "code": {
+    "coding": [
+      {
+        "system": "https://fhir.nhs.uk/CodeSystem/ServiceRequest-Code",
+        "code": "advice-and-guidance"
+      }
+    ]
+  },
+  "priority": "routine",
+  "intent": "order",
+  "status": "draft",
+  "extension": [
+    {
+      "url": "https://fhir.nhs.uk/StructureDefinition/Extension-ERS-Referral",
+      "extension": [
+        {
+          "url": "sourceSystem",
+          "valueCode": "ers"
+        }
+      ]
+    },
+    {
+      "url": "https://fhir.nhs.uk/StructureDefinition/Extension-eRS-ReferralState",
+      "extension": [
+        {
+          "url": "state",
+          "valueCoding": {
+            "system": "https://fhir.nhs.uk/CodeSystem/eRS-ReferralState",
+            "code": "INCOMPLETE",
```

Comment:
Casing. Agreed we should check [Question](https://nhsd-confluence.digital.nhs.uk/pages/viewpage.action?pageId=534350452).

This one already exists in simplifier though as we created for Patient Care.

Reply:
In Patient care, this is "https://fhir.nhs.uk/StructureDefinition/Extension-eRS-ServiceRequest-State"
should the url be consistent across the two systems?